src/encoded/types/experiment.py

```diff
@@ -431,9 +486,24 @@ class ExperimentCaptureC(Experiment):
     """The experiment class for Capture Hi-C experiments."""
     item_type = 'experiment_capture_c'
     schema = load_schema('encoded:schemas/experiment_capture_c.json')
-    embedded_list = Experiment.embedded_list + ["digestion_enzyme.name",
-                                                "targeted_regions.target.display_title",
-                                                "targeted_regions.oligo_file.href"]
+    embedded_list = Experiment.embedded_list + [
+        # Enzyme linkTo
+        'digestion_enzyme.name',
+
+        # Biofeature linkTo
+        'targeted_regions.target.feature_type',
+        'targeted_regions.target.preferred_label',
+        'targeted_regions.target.cellular_structure',
+        'targeted_regions.target.organism_name',
+        'targeted_regions.target.relevant_genes',
+        'targeted_regions.target.feature_mods',
```

Comment:
this is an object, should be `feature_mods.mod_position`, `feature_mods.mod_type` probably